www/js/pages/new-real-account-opening/components/new-account-maltainvest/new-account-maltainvest.controller.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @name new-account-maltainvest controller
+ * @author Nazanin Reihani Haghighi
+ * @contributors []
+ * @since 08/14/2016
+ * @copyright Binary Ltd
+ */
+
+(function() {
+    'use strict';
+
+    angular
+        .module('binary.pages.new-real-account-opening.components.new-account-maltainvest')
+        .controller('NewAccountMaltainvestController', NewAccountMaltainvest);
+
+    NewAccountMaltainvest.$inject = ['$scope', '$rootScope', '$state', '$timeout', '$location', 'websocketService', 'appStateService', 'accountService', '$ionicPopup', 'alertService', '$translate', 'languageService'];
```

Comment:
Please follow the role to add dependencies in order of below and in alphabetic consequence: 
1. first third native angular providers
2. then ionic providers
3. third parties providers
4. at the least our custom providers